docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,644 @@ Decision
 
 **Do not migrate!**
 
-
 subject
 =======
 
+None type
+---------
+
+Use Case
+^^^^^^^^
+
+Several elements contain unintentional null values. There are five within Tennessee Documentary History. Additional null
+subjects include vpmoore:133 and adams:76. Most of roth seems to have null mods:subject/mods:name/mods:namePart values.
+It appears we might have inserted some blank nodes using the Islandora form entry. As there is no information, these
+"values" are not used and have no true use case.
+
+Justification
+^^^^^^^^^^^^^
+
+These nodes contain no information.
+
+Xpath
+^^^^^
+
+    :code:`subject/topic[text()='']` OR
+    :code:`subject/geographic[text()='']` OR
+    :code:`subject/name/namePart[text()='']`
+
+Decision
+^^^^^^^^
+
+Do not migrate.
+
+Here's an `example record - tdh:366 <https://digital.lib.utk.edu/collections/islandora/object/tdh%3A366/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <topic/>
+    </subject>
+
+Here's an example from `roth - roth:1587 <https://digital.lib.utk.edu/collections/islandora/object/roth%3A1587/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <name authority="" valueURI="">
+            <namePart/>
+            </name>
+    </subject>
+
```

Comment:
You don't have a subjec/geographic example.  Did you want to have one?

Reply:
This is a question, but I'm good if ya'll are good.

Reply:
Thanks for this. I think that we likely should address this for all top level elements instead of doing it individually for each one simply to decrease the sheer volume of this document.

I've gone ahead and added an example for subject/geographic so that all Xpaths listed are illustrated. If we decide to remove it later that's fine, but at least we'll have it fully documented here.